src/nvim/tui/tui.c

```diff
@@ -2217,9 +2217,12 @@ static void augment_terminfo(TUIData *tui, const char *term, int vte_version, in
                                                                   "\x1b[?2004l");
   // For urxvt send BOTH xterm and old urxvt sequences. #8695
   tui->unibi_ext.enable_focus_reporting = (int)unibi_add_ext_str(ut, "ext.enable_focus",
-                                                                 rxvt ? "\x1b[?1004h\x1b]777;focus;on\x7" : "\x1b[?1004h");
-  tui->unibi_ext.disable_focus_reporting = (int)unibi_add_ext_str(ut, "ext.disable_focus",
-                                                                  rxvt ? "\x1b[?1004l\x1b]777;focus;off\x7" : "\x1b[?1004l");
+                                                                 rxvt ?
+                                                                 "\x1b[?1004h\x1b]777;focus;on\x7" :
```

Comment:
can it put the operators at the start instead of end? that is our usual style (and is preferrable imo), unless I missed something.

Reply:
Ternaries are borderline impossible to format because it violates the "inner expression indentation" rule from clint.

This question pops up pretty frequently. Should we just bite the bullet and remove the clint rule? Maybe uncrustify will gain the ability to do this in the future, if so we can start using that.

@bfredl ?

Reply:
yes, clint rule delenda est.